config/crds/all-crds.yaml

```diff
@@ -391,6 +393,7 @@ spec:
                 connect to.
               type: string
           type: object
+      type: object
```

Comment:
This appears when not using the trivialVersions controller-tools flag.
I think it's harmless, but I will double-check the CRD can be applied on k8s 1.11, just in case.

Reply:
Interesting. This reminds me that a user had to add this to fix its issue: https://github.com/elastic/cloud-on-k8s/issues/1526#issuecomment-534573668.

Reply:
Actually this comes from me removing this: https://github.com/elastic/cloud-on-k8s/pull/2480#discussion_r372430582
And applying CRDs fail on Minikube 1.11.
So I'll reintroduce the fix.